nss-emerald-wolverines/BangazonAPI
# Models only

This PR adds all models with no comments in code and without the OrderController

To test
1. git fetch --all
2. git checkout models-only
3. Checkout directory in VS
4. Confirm Order.cs has no comments
